client/app/reader/DecisionReviewer.jsx

```diff
@@ -128,6 +134,25 @@ export class DecisionReviewer extends React.PureComponent {
     ;
   }
 
+  routedPdfViewerPrototype = (props) => {
+    const { vacolsId } = props.match.params;
+
+    return <ReaderLoadingScreen
+      appealDocuments={this.props.appealDocuments}
+      annotations={this.props.annotations}
+      vacolsId={vacolsId}
+      featureToggles={this.props.featureToggles}>
+      <DocumentViewer
+        allDocuments={_.values(this.props.storeDocuments)}
+        showPdf={this.showPdf(props.history, vacolsId)}
+        history={props.history}
+        documentPathBase={`/${vacolsId}/documents`}
+        {...props}
+      />
+    </ReaderLoadingScreen>
```

Comment:
I find it easier to read when it is organized like this:
```suggestion
    return (
      <ReaderLoadingScreen
        appealDocuments={this.props.appealDocuments}
        annotations={this.props.annotations}
        vacolsId={vacolsId}
        featureToggles={this.props.featureToggles}>
          <DocumentViewer
            allDocuments={_.values(this.props.storeDocuments)}
            showPdf={this.showPdf(props.history, vacolsId)}
            history={props.history}
            documentPathBase={`/${vacolsId}/documents`}
            {...props}
          />
      </ReaderLoadingScreen>
    )
```